src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Reload/ProjectReloadInterceptor.cs

```diff
@@ -1,62 +1,62 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.ComponentModel.Composition;
+using System.Linq;
 
 using Microsoft.Build.Construction;
 
 namespace Microsoft.VisualStudio.ProjectSystem
 {
     /// <summary>
-    /// C#/VB specific project reload interceptor.
+    ///     Reloads a project if the number of configuration dimensions has changed.
```

Comment:
```suggestion
    ///     Reloads a project if its configuration dimensions change.
```

Seems that reload can occur even if the number of dimensions is the same.

Reply:
In practice this will never happen, but I've made the changes here as suggested.